File: TsogosunProfileAdminWeb/src/app/model/igt-gaming-system/playerProfile.ts
import { Address } from "./address";
import { CardNumber } from "./cardNumber";
import { Email } from "./email";
import { Employment } from "./employment";
import { Identification } from "./identification";
import { Interest } from "./interest";
import { Languages } from "./languages";
import { Name } from "./name";
import { PhoneNumber } from "./phoneNumber";
import { Restriction } from "./restriction";
import { SiteInfo } from "./siteInfo";
import { Status } from "./status";
import { userIGT } from "./userIGT";

export interface PlayerProfile {
    dateofBirth?: string | undefined;
    anniversary?: string | undefined;
    creditAccount?: string | undefined;
    gender?: string | undefined;
    pINLocked?: string | undefined;
    webEnabled?: string | undefined;
    nationality?: string | undefined;
    pipPep?: string | undefined;
    exempt?: string | undefined;
    attraction?: string | undefined;
    affiliation?: string | undefined;
    dateEnrolled?: string | undefined;
    name?: Name | undefined;
    enrolledBy?: userIGT | undefined;
    status?: Status | undefined;
    employment?: Employment | undefined;
    languages?: Languages[] | undefined;
    addresses?: Address[] | undefined;
    phoneNumbers?: PhoneNumber[] | undefined;
    identifications?: Identification[] | undefined;
    emails?: Email[] | undefined;
    cardNumbers?: CardNumber[] | undefined;
    playerRestrictions?: Restriction[] | undefined;
    interests?: Interest[] | undefined;
    siteParameters?: SiteInfo[] | undefined;
}

File: TsogosunProfileAdmin/GamingSystemIGT/Model/PlayerProfileDto/PlayerProfile.cs

using System.Collections.Generic;
using tsogosun.com.GamingSystemIGT.Model.UserDto;

namespace tsogosun.com.GamingSystemIGT.Model.PlayerProfileDto
{
    public class PlayerProfile
    {
        public string DateofBirth { set; get; }
        public string Anniversary { set; get; }
        public string CreditAccount { set; get; }
        public string Gender { set; get; }
        public string PINLocked { set; get; }
        public string WebEnabled { set; get; }
        public string Nationality { set; get; }
        public string PipPep { set; get; }
        public string Exempt { set; get; }
        public string Attraction { set; get; }
        public string Affiliation { set; get; }
        public string DateEnrolled { set; get; }
        public Name Name { set; get; }        
        public User EnrolledBy { set; get; }
        public Status Status { set; get; }
        public Employment Employment { set; get; }
        public List<Languages> Languages { set; get; }
        public List<Address> Addresses { set; get; }
        public List<PhoneNumber> PhoneNumbers { set; get; }
        public List<Identification> Identifications { set; get; }
        public List<Email> Emails { set; get; }
        public List<CardNumber> CardNumbers { set; get; }
        public List<Restriction> PlayerRestrictions { set; get; }
        public List<Interest> Interests { set; get; }
        public List<SiteInfo> SiteParameters { set; get; }
    
    }
}
